posthog/api/test/test_insight.py

```diff
@@ -266,7 +266,6 @@ def test_basic_results(self):
             ],
         )
 
-    @snapshot_postgres_queries
```

Comment:
Guessing this might be an accidental delete?

Reply:
I had to remove snapshots for this query because we're now making a separate query to the `enterprisetaggeditem` table that includes the foreign key of the related id, which makes the snapshot unstable across runs. Is there a way to this decorator ignore this case?